modules/openshift-developer-cli-pushing-the-init-image-to-a-mirror-registry-on-linux.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * openshift_cli/openshift_developer_cli/using_odo_in_a_restricted_environment/pushing-the-odo-init-image-to-a-mirror-registry/adoc
+
+[id="openshift-developer-cli-pushing-the-init-image-to-a-mirror-registry-on-linux_{context}"]
+
+= Pushing the `init` image to a mirror registry on Linux
+
+.Procedure
+
+. Use `base64` to encode your mirror registry's root certification authority (CA) content:
```

Comment:
to encode the root certification authority (CA) content of your mirror registry
Avoid anthropomorphism.